sdk/metricsadvisor/Azure.AI.MetricsAdvisor/tests/MetricsAdvisorAdministrationClient/AnomalyDetectionConfigurationLiveTests.cs

```diff
@@ -14,10 +14,6 @@ namespace Azure.AI.MetricsAdvisor.Tests
 {
     public class AnomalyDetectionConfigurationLiveTests : MetricsAdvisorLiveTestBase
     {
-        private const string MetricName = "metric";
-        private const string DimensionNameA = "dimensionA";
-        private const string DimensionNameB = "dimensionB";
```

Comment:
Moving this and dimension key validation methods to the base class to share code across test files.